training/distributed_training/tensorflow/data_parallel/efficientnet/EfficientNet_Notebook.ipynb

```diff
@@ -0,0 +1,368 @@
+{
```

Comment:
<p>Use link text rather than the raw efficientnet URL. In general throughout the notebook, avoid raw links. It's messy and harder to read.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3060/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>